static/app/views/organizationLayout.tsx

```diff
@@ -0,0 +1,81 @@
+import {Fragment} from 'react';
+import styled from '@emotion/styled';
+
+import {Alert} from 'sentry/components/alert';
+import HookOrDefault from 'sentry/components/hookOrDefault';
+import LoadingError from 'sentry/components/loadingError';
+import LoadingTriangle from 'sentry/components/loadingTriangle';
+import SentryDocumentTitle from 'sentry/components/sentryDocumentTitle';
+import Sidebar from 'sentry/components/sidebar';
+import {ORGANIZATION_FETCH_ERROR_TYPES} from 'sentry/constants';
+import {t} from 'sentry/locale';
+import OrganizationStore from 'sentry/stores/organizationStore';
+import {useLegacyStore} from 'sentry/stores/useLegacyStore';
+import {space} from 'sentry/styles/space';
+
+interface OrganizationLayoutProps {
+  /**
+   * Render the sidebar when possible if enabled
+   */
+  includeSidebar: boolean;
+  children?: React.ReactNode;
+}
+
+const OrganizationHeader = HookOrDefault({
+  hookName: 'component:organization-header',
+});
+
+/**
+ * Renders the organization page layout
+ */
+function OrganizationLayout({includeSidebar, children}: OrganizationLayoutProps) {
+  const {organization, loading, error, errorType} = useLegacyStore(OrganizationStore);
+
+  if (loading) {
+    return <LoadingTriangle>{t('Loading data for your organization.')}</LoadingTriangle>;
+  }
+
+  const sidebar = includeSidebar ? (
+    <Sidebar organization={organization ?? undefined} />
```

Comment:
Can org be null? If it cant then this isn't doing anything as undefined ?? undefined = undefined

Reply:
yeah it is typed as null